invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -302,7 +302,8 @@ def _format_thesis(thesis):
     # Retrieve publishing related custom fields
     journal = field.get("journal:journal")
     imprint = field.get("imprint:imprint")
-    thesis = field.get("thesis:thesis")
+
```

Comment:
I'd add a comment here that "thesis:university" is a deprecated field and will be removed in a future release.